governance-server/src/util.js

```diff
@@ -3,6 +3,8 @@ const blake2b = require('blake2b');
 const Crypto = require('@aeternity/aepp-sdk').Crypto;
 const BigNumber = require('bignumber.js');
 
+BigInt.prototype.toJSON = function () { return Number(this) }
```

Comment:
needed as we can't `JSON.stringify` with bigint, currently all data is optionally cached using strings. This will break for larger numbers but hasn't been a problem in the past. A refactor wouldn't be trivial